src/shutdown/umount.c

```diff
@@ -761,10 +780,19 @@ static int mount_points_list_umount(MountPoint **head, bool *changed, bool last_
 
                         xsprintf(newpath, "/run/shutdown/mounts/%08" PRIu64, random_u64());
 
-                        if (is_dir(m->path, true))
-                                (void) mkdir_p(newpath, 0700);
-                        else
-                                (void) touch_file(newpath, true, USEC_INFINITY, UID_INVALID, GID_INVALID, 0700);
+                        if (is_dir(m->path, true)) {
+                                r = mkdir_p(newpath, 0700);
+                                if (r < 0) {
+                                        log_full_errno(last_try ? LOG_ERR : LOG_INFO, r, "Could not create directory %s: %m", newpath);
+                                        continue;
+                                }
+                        } else {
+                                r = touch_file(newpath, true, USEC_INFINITY, UID_INVALID, GID_INVALID, 0700);
+                                if (r < 0) {
+                                        log_full_errno(last_try ? LOG_ERR : LOG_INFO, r, "Could not create file %s: %m", newpath);
+                                        continue;
+                                }
+                        }
```

Comment:
this part of the change ended up in the wrong commit i guess?

Reply:
It was.